data/waellerbe.json

```
@@ -1,11 +1,17 @@
 {
   "name": "William A Ellerbe",
-  "bio": "Prompt Engineering Consultant with a passion for Open Source projects",
-  "links": [
+  "bio": "Open Source | Prompt Engineering | Proud member of **EddieHub**",
+"links": [
 {
+      "group": "Socials Primary",
       "name": "Follow me on LinkedIn",
       "url": "https://www.linkedin.com/in/williamaellerbe/",
       "icon": "FaLinkedin"
-    }
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/waellerbe",
+      "icon": "FaGithub",
```

removing comma and adding closing bracket:
```suggestion
      "icon": "FaGithub"
   }
```